jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java

```diff
@@ -59,12 +61,14 @@ public RemoteReporter(final Sender sender, int flushInterval, int maxQueueSize,
     commandQueue = new ArrayBlockingQueue<>(maxQueueSize);
 
     // start a thread to append spans
+    long serialNumber = nextSerialNumber.getAndIncrement();
```

Comment:
Seemed like a good idea to ensure thread name uniqueness, but you're right, it's not necessary.  This is my first pull request on GitHub: do I make the change in my repo then push, or do you do it here directly?